shard-operator-parent/shard-operator-v2/src/test/java/com/redhat/service/smartevents/shard/operator/v2/ManagedProcessorServiceTest.java

```diff
@@ -0,0 +1,81 @@
+package com.redhat.service.smartevents.shard.operator.v2;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.redhat.service.smartevents.infra.v2.api.models.OperationType;
+import com.redhat.service.smartevents.infra.v2.api.models.dto.ProcessorDTO;
+import com.redhat.service.smartevents.shard.operator.v2.resources.ManagedProcessor;
+import com.redhat.service.smartevents.shard.operator.v2.utils.Fixtures;
+import com.redhat.service.smartevents.shard.operator.v2.utils.V2KubernetesResourcePatcher;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.kubernetes.client.WithOpenShiftTestServer;
+
+@QuarkusTest
+@WithOpenShiftTestServer
+public class ManagedProcessorServiceTest {
+
+    @Inject
+    ManagedProcessorService managedProcessorService;
+
+    @Inject
+    V2KubernetesResourcePatcher kubernetesResourcePatcher;
+
+    @BeforeEach
+    public void beforeEach() {
+        kubernetesResourcePatcher.cleanUp();
+    }
+
+    @Test
+    public void TestCreateManagedProcessor() {
+        // setup
+        ProcessorDTO processorDTO = Fixtures.createProcessor(OperationType.CREATE);
+
+        // test
+        managedProcessorService.createManagedProcessor(processorDTO);
+
+        // assert
+        List<ManagedProcessor> managedProcessorList = managedProcessorService.fetchAllManagedProcessors();
+        Assertions.assertThat(managedProcessorList.size()).isEqualTo(1);
+    }
+
+    @Test
+    public void TestDeleteManagedProcessor() {
```

Comment:
If I was being picky I'd say the method name should be camel-case not pascal-case.